hibernate-core/src/main/java/org/hibernate/dialect/function/TimestampdiffFunction.java

```diff
@@ -58,7 +59,7 @@ public void render(
 				TypeConfiguration.getSqlTemporalType( to.getExpressionType() )
 		);
 
-		new PatternRenderer( pattern ).render( sqlAppender, arguments, walker );
+		new PatternRenderer( pattern, SqlAstNodeRenderingMode.DEFAULT ).render( sqlAppender, arguments, walker );
```

Comment:
Since you almost always seem to pass `SqlAstNodeRenderingMode.DEFAULT` to `new PatternRenderer()`, wouldn't it make more sense to overload the constructor?